contracts/src/lib.cairo

```diff
@@ -3,7 +3,7 @@
 mod ocr2;
 mod libraries;
 mod emergency;
-mod multisig;
```

Comment:
temporarily comment this out because there are compile errors

Reply:
Can you leave it in but commented out? It seems to be replaced here

Reply:
Compilation should be fixed now, just need to rebase

Reply:
Do the tests still fail for you? They pass on the cairo-1.0 branch with the new CI check